External configuration services of logger for Iron Release. (#3478)
ros2/ros2_documentation: source/Releases/Release-Iron-Irwini.rst

See the tracking issue at https://github.com/ros2/rmw/issues/330 for more information.

External configuration services of loggers
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> It is now possible to configure the logger levels from remote node via service.
> When ``enable_logger_service`` option is enabled on node creation, ``set_logger_levels`` and ``get_logger_levels`` services will be available under the node.
> These services allow user application to change the logging level of logger from the remote node.
> 
> Be advised that ``enable_logger_service`` is disabled by default, user needs to enable this option on node creation.

Reviewer: ```suggestion
It is now possible to configure node logger levels remotely via a service.
When the ``enable_logger_service`` option is enabled during node creation, the ``set_logger_levels`` and ``get_logger_levels`` services will be available.

Be advised that the ``enable_logger_service`` option is disabled by default, so the user needs to enable this option on node creation.
```

Author: fixed by https://github.com/ros2/ros2_documentation/pull/3478/commits/6ef4d49d6ed81bbaf9b4283c876e393ff251af7a

> There are examples showing how to do this in https://github.com/ros2/demos/tree/{DISTRO}/demo_nodes_cpp/src/logging (C++) and https://github.com/ros2/demos/blob/{DISTRO}/demo_nodes_py/demo_nodes_py/logging (Python).

Author: we would probably want to point out the specific files as demonstration, but that brings `file unavailable` for humble and foxy since this is new feature available on iron or later.

Reviewer: I might make a suggestion here in that we leave out this sentence for now.  If we do that, then we can merge this release note right now (we don't need to wait for https://github.com/ros2/demos/pull/611 to be merged), and we avoid the problem you point out here[1].  Then when we eventually merge the demo, we can add just this line back (and have it point to Iron).  What do you think?

[1] I realize that we actually have this problem in a couple of other places as well.  I'll update them to fix it.

Author: makes sense to me.

> see https://github.com/ros2/ros2/issues/1355 for more information.

Reviewer: ```suggestion
See https://github.com/ros2/ros2/issues/1355 for more information.
```

Author: fixed by https://github.com/ros2/ros2_documentation/pull/3478/commits/6ef4d49d6ed81bbaf9b4283c876e393ff251af7a
